blockchain/My-Wallet-V3-iOS
# feat(HB): Hooking up Exchange Locked Screen - IOS-1210

## Objective

Hooks up the `Exchange Locked` screen and handles the empty state of the `Exchange List` screen.

## Description

The `Exchange List` screen (Order History) will now show a loading indicator initially. The `Create New Order` button is always visible regardless whether or not trades have been placed. The user must tap this button in order to create a new trade. 

After submitting a trade, we pop back to the rootVC and then present the `Exchange Locked` screen modally. 

## How to Test

1. Build an run
2. Copy your BTC address
3. Enter into the settings debug menu
4. Go to the `Exchange` flow
5. Observer the `Exchange List` screen
6. Create a new exchange
7. Submit the exchange
8. View the `Exchange Locked` screen
9. Tap `Done` and you should be back at the `Exchange List` screen.

## Screenshot/Design assessment

See ios-dev channel for demo. 

## Merge Checklist
